src/content/docs/distributed-tracing/concepts/introduction-distributed-tracing.mdx

```diff
@@ -0,0 +1,143 @@
+
+For example, let's say that you're in a situation where a slow-running request affects the experience of a set of customers:
+
+* The request is distributed across multiple microservices and serverless functions. 
+* Several different teams own and monitor the various services that are involved in the request.
+* None of the teams have reported any performance issues with their microservices. 
+
+Without a way to view the performance of the entire request across the different services, it’s nearly impossible to pinpoint where and why the high latency is occurring and which team should address the issue. 
+
+## Instrumentation: The key to distributed tracing
+
+Distributed tracing starts with the instrumentation of your services to enable data collection and correlation across the entire distributed system. Instrumention means either manually adding code to services or installing agents that automatically track trace data. 
+
+Many of our New Relic solutions automatically instrument your services for a large number of programming languages and frameworks. You can also use open source tools and open instrumentation standards to instrument your environment. OpenTelemetry, part of the [Cloud Native Computing Foundation (CNCF)](https://www.cncf.io/), is becoming the one standard for open source instrumentation and telemetry collection. 
+
+## What you can do with distributed tracing and New Relic
+
+After the data is collected, you can visualize it to see service dependencies, performance, and any anomalous events such as errors or unusual latency. Here are some examples of what you can do with your data:
+
+<table>
+  <thead>
+    <tr>
+      <th style={{ width: "250px" }}>
+        What you can do
+      </th>
+
+      <th>
+        Description
+      </th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <tr>
+      <td>
+        [Detect anomalous spans](/docs/apm/distributed-tracing/ui-data/understand-use-distributed-tracing-data#anomalous-spans)
+      </td>
+
+      <td>
+        Spans that are slow in comparison to typical behavior are marked as anomalous, with charts comparing them to typical performance.
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        [See your errors and logs](/docs/understand-dependencies/distributed-tracing/ui-data/understand-use-distributed-tracing-ui#view-your-logs)
+      </td>
+
+      <td>
+        Frontend and backend errors appear right in the context of your traces. Everything you need to troubleshoot is in one place.
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        [Filter results](/docs/apm/distributed-tracing/ui-data/understand-use-distributed-tracing-data)
+      </td>
+
+      <td>
+        You can filter charts using many data points, so you can analyze trace data in different ways.
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        [Customize queries and dashboards](/docs/insights/use-insights-ui/manage-dashboards/create-edit-copy-insights-dashboards)
```

Comment:
I'd prob link this to 'INtro to NRQL' doc instead. For getting started building custom queries, charts.

Reply:
@zuluecho9, I switched this to NRQL.